README.md

```diff
@@ -1,7 +1,71 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Sowon Ham . Jarvis Consulting
+
+I am a recent graduate of University of Toronto - St. George Campus and I studied statistics and mathematics in my degree. Due to my passion in the DA/BA/BSA field I pursued to a profession that suits my strength and my likings. Jarvis Consulting Group is the first place that I was able to thoroughly communicate with and resulted in a very positive manner to work here as one of the employee.
```

Comment:
I am a recent graduate from the University of Toronto - St. George Campus, and I studied statistics and mathematics in my degree. Due to my passion in the data and business analytics fields, I pursued a profession that suits my strengths and interests.

Jarvis is the first place that I was able to thoroughly communicate with and resulted in a very positive manner to work here as one of the employee.
		

It is such an honor to be working in a field that is continuously growing every day and see everyone including myself studying new subjects every day. 

Particularly, while I was studying in my field the process of gathering data, building statistical models, and analyzing through mathematical methods fascinated me to a great extent. I wish to pursue in this field to continuously learn and apply old and new methods of data analysis to play with data.

Background 
Jarvis -> technical skills, SDLC, Agile, soft skills 
School study -> passion + technical skills + abckground
Passion -> soft skills

		
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee